### PR TITLE
Add information about credentials

### PR DESCRIPTION
**Pull request by aivus**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -40,6 +40,12 @@ Your machine is now running!
 ### Access to your box
 Find out the IP of your guest machine. Go to http://find.synology.com to connect to the virtual Synology box, use the Synology Assistant (http://www.synology.com/en-us/support/download), or use something else.
 
+**Credentials:**
+
+Login: vagrant
```
Comment (sebask) [inline README.md]:
```suggestion
Username: `vagrant`
```

File: README.md [hunk 2]
```diff
@@ -40,6 +40,12 @@ Your machine is now running!
 ### Access to your box
 Find out the IP of your guest machine. Go to http://find.synology.com to connect to the virtual Synology box, use the Synology Assistant (http://www.synology.com/en-us/support/download), or use something else.
 
+**Credentials:**
```
Comment (sebask) [inline README.md]:
```suggestion
**Credentials preconfigured Vagrant user:**
```

File: README.md [hunk 3]
```diff
@@ -40,6 +40,12 @@ Your machine is now running!
...
+
+Login: vagrant
+
```
Comment (sebask) [inline README.md]:
```suggestion
```

File: README.md [hunk 4]
```diff
@@ -40,6 +40,12 @@ Your machine is now running!
...
+Password: vagrant
```
Comment (sebask) [inline README.md]:
```suggestion
Password: `vagrant`
```

**Review by sebask — CHANGES_REQUESTED** [review]

**Comment by aivus** [comment]
@sebask done

**Review by sebask — APPROVED** [review]

**Pull request merged**